src/techCompanies/orderByAlphabeth/l.js

```diff
@@ -40,8 +40,15 @@ export default {
       companyName: "Lightech-I.T Firm",
       companyTwitterHandle: "Lightech_ITFirm",
       foundersLinkedin: "https://www.linkedin.com/in/noble-amaechi-63b13986" ,
-      industry: "Ul/Ux & Product Design ",
+      industry: "UI/Ux & Product Design ",
       website: "https://www.lightech.com.ng/",
     },
+    {
+      companyName: "Lihon Energy",
+      companyLinkedinHandle: "Lihon Energy",
+      foundersLinkedin: "Hakeem Disu" ,
```

Comment:
Adding a `companyLinkinHandle` and `foundersLinkedin` does not conform to the schema of the other companies and that can be consequential to the proper functioning of the site that depends on it. https://tech-companies-in-africa-ph1srbc96-osca-ado-ekiti.vercel.app/